charts/docker-hub-rate-limit-exporter-chart/Chart.yaml

```diff
@@ -16,7 +16,7 @@ type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 0.2.7 
+version: 0.2.6
```

Comment:
Why not use 0.2.8 instead here? :-)

Reply:
Resolved with https://github.com/viadee/docker-hub-rate-limit-exporter/pull/22/commits/10538c7cd82a9c566adc866005ca713cbf41a2c2